spec/ics-23-standard-accumulator/README.md

```diff
@@ -10,14 +10,15 @@ modified: 2019-04-25
 
 # Synopsis
 
-An *accumulator*, or *cryptographic accumulator*, is a construction that produces a succinct, binding commitment to a set or indexed vector of elements and short membership and/or non-membership proofs for any element in the set or vector.
-This specification enumerates the functions and properties required of accumulator constructions used in the IBC protocol.
+An *accumulator*, or *cryptographic accumulator*, is a construction that produces a succinct, binding commitment to an indexed vector of elements and short membership and/or non-membership proofs for any indicies & elements in the vector.
```

Comment:
why refer to accumulators at all, couldn't everything just be described around vector commitments?

Reply:
I suppose but "accumulator" is more likely to be a known term - vector commitments are considered to be a kind of accumulator, right?

Reply:
I'm not an expert by any means on accumulators or vector commitments, but I typically see them refferred to distinctly, not as vector commitments being a subset of the other. Formally one is not the subset of the other, they do different things. (For instance, look at page 1 of https://eprint.iacr.org/2018/1188.pdf)

I don't think accumulators are necessarily a more known term. I am confused as to the title of this ICS. Perhaps the title should be something more general (proof system API requirements?), and therein in can define vector commitments.

Reply:
"State Commitments" perhaps?

Reply:
We can just call them vector commitments if that's all we'll use, it seems so for now.

Reply:
Sounds great to use vector commitments for now!